clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/op/manifest/KubernetesPatchManifestOperation.java

```diff
@@ -48,17 +48,17 @@
   private final KubernetesV2Credentials credentials;
   private final KubernetesResourcePropertyRegistry registry;
   private final String accountName;
+  private final ObjectMapper objectMapper;
   private static final String OP_NAME = "PATCH_KUBERNETES_MANIFEST";
 
-  @Autowired
-  private ObjectMapper objectMapper;
```

Comment:
I don't see how this would even work. This class is not created in the spring context and therefore will not have any beans autowired into it.